snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml

```diff
@@ -1,14 +1,30 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  labels:
+    app.kubernetes.io/instance: snyk-monitor
```

Comment:
the constant "0.0.0" appears in multiple places here, is it supposed to stay "0.0.0" when we build the operator?

Reply:
No, we replace it for the current `kubernetes-monitor` version. You can take a look [here](https://github.com/snyk/kubernetes-monitor/blob/6dc63bc45bc8e3cbc58c3b67b77652e1cc2357d0/scripts/operator/package-operator.sh#L33).